usaspending_api/api_contracts/contracts/award/idv/FundingAccounts.md

```diff
@@ -42,5 +42,8 @@ This endpoint returns a list of federal accounts under a given IDV.
 
 ## AccountListing (object)
 + total_transaction_obligated_amount: 1234.56 (required, number)
-+ federal_acount: `075-1301` (required, string)
-+ account_title: `National Institutes of Health` (required, string)
\ No newline at end of file
++ federal_account: `075-1301` (required, string)
++ account_title: `Bureau of Consumer Financial Protection Fund` (required, string)
++ abbreviation: `NIH` (required, string)
```

Comment:
Using clear names would be better:
- funding_agency_abbreviation
- funding_agency_name
- funding_agency_id